charts/kyverno/values.yaml

```diff
@@ -503,6 +503,8 @@ tufRootMountPath: /.sigstore
 grafana:
   # -- Enable grafana dashboard creation.
   enabled: false
+  # -- Configmap name default is the kyverno configmap name with "-grafana" appended
+  configmapName: ~
```

Comment:
How about setting the template here directly ?

Reply:
Yeah I mean putting the template in the values directly and calling `tpl`.
We do this in a couple of places already.

Reply:
Yes, exactly that. IMHO using an helper is good when we allow end users to bring their own thing, it's not the case here.

Reply:
@Alvaro-Campesino I pushed a commit, WDYT ?

Reply:
If you prefer `configMapName` go ahead and change it.